### PR TITLE
add 52592 fix control

### PR DESCRIPTION
**Pull request by dbsid**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add doc the for new fix control 52592

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: optimizer-fix-controls.md [hunk 1]
```diff
@@ -26,6 +26,15 @@ SET SESSION tidb_opt_fix_control = '44262:ON,44389:ON';
...
 
+### [`52592`](https://github.com/pingcap/tidb/issues/52592) <span class="version-mark">New in v8.4.0</span>
+
+- Default value: `OFF`
+- Possible values: `ON`, `OFF`
+- Controls whether to allow data retrieval via Get or BatchGet. If set to `ON`, the optimizer will force the selection of Coprocessor, while both Get and BatchGet will be disabled. Get and BatchGet do not support column projection, and in some scenarios, their execution cost is higher than Coprocessor requests, setting this to `ON` will result in better performance. The following are common cases where forcing Coprocessor selection is recommended:
+	- Wide tables with many columns, where applications only query a smallsubset of columns.
+	- Tables containing JSON columns with large JSON values, where queries either do not retrieve the JSON columns or only extract small portions of the JSON content.
+	 
+
```
Comment (winoros) [inline optimizer-fix-controls.md:37]: The order is alphabetic.
We'd better add it at the end of the chapter.

File: optimizer-fix-controls.md [hunk 2]
```diff
@@ -71,6 +71,14 @@ SET SESSION tidb_opt_fix_control = '44262:ON,44389:ON';
...
 - `0` means to disable this heuristic strategy.
 
+### [`52592`](https://github.com/pingcap/tidb/issues/52592) <span class="version-mark">New in v8.4.0</span>
+
+- Default value: `OFF`
+- Possible values: `ON`, `OFF`
+- Controls whether to allow data retrieval via Get or BatchGet. If set to `ON`, the optimizer will force the selection of Coprocessor, while both Get and BatchGet will be disabled. Get and BatchGet do not support column projection, and in some scenarios, their execution cost is higher than Coprocessor requests, setting this to `ON` will result in better performance. The following are common cases where forcing Coprocessor selection is recommended:
+	- Wide tables with many columns, where applications only query a smallsubset of columns.
+	- Tables containing JSON columns with large JSON values, where queries either do not retrieve the JSON columns or only extract small portions of the JSON content.
```
Comment (qiancai) [inline optimizer-fix-controls.md:80]:
```suggestion
- This variable controls whether to disable the `Point Get` and `Batch Point Get` operators for query execution. The default value `OFF` means that `Point Get` and `Batch Point Get` can be used for query execution. If set to `ON`, the optimizer disables `Point Get` and `Batch Point Get`, forcing the selection of Coprocessor for query execution.
- `Point Get` and `Batch Point Get` do not support column projection (that is, they cannot return only a subset of the columns), so in some scenarios, their execution efficiency might be lower than that of the Coprocessor, and setting this variable to `ON` can improve query performance. The following are recommended scenarios for setting this variable to `ON`:

    - Queries on wide tables with many columns, where only a few columns need to be retrieved.
    - Queries on tables with large JSON values, where the entire JSON column does not need to be retrieved, or only a small portion of the JSON column needs to be retrieved.
```

**Review by Oreoxmt — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

File: optimizer-fix-controls.md [hunk 3]
```diff
@@ -71,6 +71,16 @@ SET SESSION tidb_opt_fix_control = '44262:ON,44389:ON';
...
+### [`52592`](https://github.com/pingcap/tidb/issues/52592) <span class="version-mark">New in v8.4.0</span>
+
+- Default value: `OFF`
+- Possible values: `ON`, `OFF`
+- This variable controls whether to disable the `Point Get` and `Batch Point Get` operators for query execution. The default value `OFF` means that `Point Get` and `Batch Point Get` can be used for query execution. If set to `ON`, the optimizer disables `Point Get` and `Batch Point Get`, forcing the selection of Coprocessor for query execution.
+- `Point Get` and `Batch Point Get` do not support column projection (that is, they cannot return only a subset of the columns), so in some scenarios, their execution efficiency might be lower than that of the Coprocessor, and setting this variable to `ON` can improve query performance. The following are recommended scenarios for setting this variable to `ON`:
+
+    - Wide tables with many columns, where only a few columns are queried.
+    - Tables with large JSON values, where the JSON column is not queried, or only a small portion of the JSON column is queried.
+
```
Comment (qiancai) [inline optimizer-fix-controls.md:83]:
```suggestion

```

File: optimizer-fix-controls.md [hunk 4]
```diff
@@ -71,6 +71,15 @@ SET SESSION tidb_opt_fix_control = '44262:ON,44389:ON';
...
 
+### [`52592`](https://github.com/pingcap/tidb/issues/52592) <span class="version-mark">New in v8.4.0</span>
+
+- Default value: `OFF`
+- Possible values: `ON`, `OFF`
+- This variable controls whether to disable the `Point Get` and `Batch Point Get` operators for query execution. The default value `OFF` means that `Point Get` and `Batch Point Get` can be used for query execution. If set to `ON`, the optimizer disables `Point Get` and `Batch Point Get`, forcing the selection of Coprocessor for query execution.
+- `Point Get` and `Batch Point Get` do not support column projection (that is, they cannot return only a subset of the columns), so in some scenarios, their execution efficiency might be lower than that of the Coprocessor, and setting this variable to `ON` can improve query performance. The following are recommended scenarios for setting this variable to `ON`:
+
+    - Wide tables with many columns, where only a few columns are queried.
+    - Tables with large JSON values, where the JSON column is not queried, or only a small portion of the JSON column is queried.
```
Comment (qiancai) [inline optimizer-fix-controls.md:82]:
```suggestion
    - Tables with large JSON values, where the JSON column is not queried, or only a small portion of the JSON column is queried.

```

**Pull request merged**